app/controllers/users/sessions_controller.rb

```diff
@@ -1,34 +1,9 @@
 class Users::SessionsController < Devise::SessionsController
   before_action :configure_sign_in_params, only: [:create]
 
-  # GET /resource/sign_in
-  # def new
-  #   super
-  # end
-
-  # POST /resource/sign_in
-  # def create
-  #   super
-  # end
-
-  # DELETE /resource/sign_out
-  # def destroy
-  #   super
-  # end
-
-  # protected
-
   # If you have extra params to permit, append them to the sanitizer.
-  # def configure_sign_in_params
-  #   devise_parameter_sanitizer.permit(:sign_in, keys: [:attribute])
-  # end
+   def configure_sign_in_params
```

Comment:
In case we want to support logging in by ```username``` in the future. It doesn't hurt having it there but it helps future teams be less confused I think if they want to add more params for signing in.